src/ios/CDVStatusBar.m

```diff
@@ -186,6 +192,18 @@ - (void)updateIsVisible:(BOOL)visible {
     [self.commandDelegate sendPluginResult:result callbackId:_eventsCallbackId];
 }
 
+- (void)disableViewportFitForiOS12:(BOOL)disable {
```

Comment:
No longer required since iOS 13 is the minimum supported version? Or does this mean iOS12 and later?